packages/example/src/mocks/handlers/search.ts

```diff
@@ -68,6 +68,9 @@ export default [
             title.toLocaleLowerCase().includes(q.toLocaleLowerCase())
           )
         : items;
-    return response(context.status(200), context.json(searchResults));
+
+    return HttpResponse.json(searchResults, {
+      status: 200,
```

Comment:
I believe the 'extra' `status: 200` is not needed here, since it is the default return status ??

Reply:
Good call, I didn't realize this initially. I meant to go back and clean this up, but was super pressed for time. Thank for pointing this out!